rfc/rfc-44/rfc-44.md

```diff
@@ -0,0 +1,156 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# RFC-44: Hudi Connector for Presto
+
+## Proposers
+
+- @7c00
+
+## Approvers
+
+- @
+
+## Status
+
+JIRA: [HUDI-3210](https://issues.apache.org/jira/browse/HUDI-3210)
+
+> Please keep the status updated in `rfc/README.md`.
+
+## Abstract
+
+The support for querying Hudi tables in Presto is provided by Presto Hive connector. The implementation is built on the
+InputFormat interface from hudi-hadoop-mr module. This approach has known performance and stability issues. It's also
```

Comment:
actually the real issue has been maintainability i.e code often gets changed in Presto without us tracking/following with side-effects.